app/javascript/dashboard/components/ChatList.vue

```diff
@@ -188,6 +194,42 @@ export default {
     });
   },
   methods: {
+    handleKeyEvents(e) {
+      if (hasPressedShiftAndJKey(e)) {
+        const allConversation = this.$refs.activeConversation.querySelectorAll(
+          'div.conversation'
+        );
+        const activeConversation = this.$refs.activeConversation.querySelector(
+          'div.conversation.active'
+        );
+        const activeConversationIndex = [...allConversation].indexOf(
+          activeConversation
+        );
+        if (activeConversationIndex <= 0) {
+          allConversation[0].click();
+        }
+        if (activeConversationIndex >= 1) {
+          allConversation[activeConversationIndex - 1].click();
+        }
+      }
+      if (hasPressedShiftAndKKey(e)) {
+        const allConversation = this.$refs.activeConversation.querySelectorAll(
+          'div.conversation'
```

Comment:
This selector needs to specifically target the conversations list otherwise previous conversations in sidebar will also be included. See other places also to make css selector path specific